package/yast2-network.changes

```diff
@@ -1,4 +1,41 @@
 -------------------------------------------------------------------
+Mon Apr 27 07:06:18 UTC 2015 - mfilka@suse.com
+
+- bnc#912904 
+  - do not crash when adding new device and proposed configuration
+    name is changed to new one
+- bnc#914833
+  - do not crash when renaming virtual device via hardware tab
+- bnc#910337
+  - initialize configuration name in Hardware tab correctly. It
+    means according device type. Also uses custom name if user
+    used nonstandard one.
+- bnc#905738
+  - udev rules provided by AutoYaST profile gets applied
+- bnc#918356
+  - populate bond slave candidates list with proper device names
+- 3.1.114
```

Comment:
This is a problem, we already have this version (had this version in the past and it was different I guess). Maintenance branches must use X.Y.Z.A versioning instead of X.Y.Z (as used in development).